tlscanary/modes/regression.py

```diff
@@ -91,12 +94,12 @@ def run(self):
         rldb = rl.RunLogDB(self.args)
         log = rldb.new_log()
         log.start(meta=meta)
-        progress = pr.ProgressTracker(total=len(self.sources), unit="hosts", average=30*60.0)
+        progress = pr.ProgressTracker(total=len(self.sources), unit="hosts", average=30 * 60.0)
```

Comment:
I find whitespace around operators weird when there is no whitespace around the equal sign as well. Since whitespace around the equal sign for default parameters violates PEP8, I strongly prefer local consistency.